### PR TITLE
RIA-3722 edit listing for remote hearing

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-3722](https://tools.hmcts.net/jira/browse/RIA-3722)

### Change description ###

See Jira

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-3722-edit-case-listing-notification-remote-hearing-no-change.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        "subject": "Immigration and Asylum appeal: change to Hearing requirements",
+        "body": [
+          "PA/12345/2019",
+          "LP/12345/2019",
+          "A1234567",
+          "Talha Awan",
+          "{$iaExUiFrontendUrl}",
+          "Remote hearing",
+          "3 May 2019",
+          "3 May 2019",
```
Comment (mkayad) [inline src/functionalTest/resources/scenarios/RIA-3722-edit-case-listing-notification-remote-hearing-no-change.json:92]: Same thing as above

**Review by mkayad — DISMISSED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**